### ISSUE TITLE
libtorrent 2.0.4 with mingw64

### ISSUE THREAD

**Issue by 3unjee** [arvidn/libtorrent#6274]
Greetings libtorrent enthuisiasts and peer to peer advocates among the masses, 

For some reason, when running the following on Windows:
`b2 -j4 toolset=gcc cxxstd=14 variant=release link=shared threading=multi install --prefix="$PWD/build"`
I get this:
```
building boost from source directory:  C:\dev\workspace\libtorrent\boost
Performing configuration checks

    - default address-model    : 64-bit
    - default architecture     : x86
error: Unable to find file or target named
error:     'libtorrent-rasterbar.pc'
error: referred to from project at
error:     '.'
```
What's weird is when I actually call that a second thime:
`b2 -j4 toolset=gcc cxxstd=14 variant=release link=shared threading=multi install --prefix="$PWD/build"`

The compilation actually works.

**Comment by arvidn** [comment]
yeah, the creation of the `libtorrent-rasterar.pc` file is a bit of a hack. I should have realized the ordering isn't guaranteed to work right.

**Comment by FranciscoPombal** [comment]
Works fine with CMake, btw.

```bash
cmake -G Ninja -B cmake-build-dir -D CMAKE_BUILD_TYPE=Release -D deprecated-functions=OFF -D developer-options=ON -D CMAKE_INSTALL_PREFIX="./install_dir"
cmake --build cmake-build-dir
cmake --install cmake-build-dir
```

**Comment by 3unjee** [comment]
> yeah, the creation of the `libtorrent-rasterar.pc` file is a bit of a hack. I should have realized the ordering isn't guaranteed to work right.

To be fixed with an upcoming version ?
